Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile
+
+FROM circleci/python:3.9.2
```

Comment:
I think we could use here "normal" python image (It is not needed for local development to pull entire new python image from circleCI). We will duplicate this step also in `config.yml` via `docker: ... ` paragraph, and there we will use cicrleCi'ish images.

Reply:
Done